yarn-project/prover-node/src/job/epoch-proving-job.ts

```diff
@@ -154,14 +155,18 @@ export class EpochProvingJob implements Traceable {
       this.metrics.recordProvingJob(executionTime, timer.ms(), epochSizeBlocks, epochSizeTxs);
     } catch (err: any) {
       if (err && err.name === 'HaltExecutionError') {
-        this.log.warn(`Halted execution of epoch ${epochNumber} prover job`, { uuid: this.uuid, epochNumber });
+        this.log.warn(`Halted execution of epoch ${epochNumber} prover job`, {
+          uuid: this.uuid,
+          epochNumber,
+          details: err.message,
+        });
         return;
       }
       this.log.error(`Error running epoch ${epochNumber} prover job`, err, { uuid: this.uuid, epochNumber });
       this.state = 'failed';
     } finally {
       clearTimeout(this.deadlineTimeoutHandler);
-      await this.cleanUp(this);
+      await this.epochCheckPromise?.stop();
```

Comment:
do we want to be removing `cleanup` here?

Reply:
I see further down it's no longer passed, should we remove it completely from the constructor signature?

Reply:
Not sure I follow, I removed it from the proving-job constructor. Or do you mean removing from somewhere else?

Reply:
yep nvm was looking at the wrong place 👍